addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -80,11 +80,10 @@ const Link = Widget.extend({
                 }
                 $node = $node.parent();
             }
-            const textLink = this.$link[0] || this.data.range.cloneContents();
-            const [encodedText, images] = this._encodeNodeToText(textLink);
-            this.data.originalText = this._decodeText(encodedText, images);
-            this.data.content = encodedText;
-            this.data.images = images;
+            const linkNode = this.$link[0] || this.data.range.cloneContents();
+            const linkText = linkNode.innerHTML || linkNode.textContent;
+            this.data.content = linkText.replace(/[ \t\r\n]+/g, ' ');
```

Comment:
Is it actually needed? :thinking: If yes, maybe worth a comment? Also is not `\s+` more accurate and also simpler?

Reply:
To be honest, I don't know. I saw that it was done in the `else` clause and inside the `encodeTextNode` conversion, so basically it was done in all cases so I figured keeping it wouldn't hurt, while removing it might introduce a weird bug somewhere down the line.

Reply:
I guess it should not hurt indeed...